Makefile

```diff
@@ -24,3 +24,10 @@ HEROKU_APP_US = $(SERVICE_SYSTEM_CODE)-us
 GRAFANA_DASHBOARD = $(SERVICE_SYSTEM_CODE)
 
 export GITHUB_RELEASE_REPO := Financial-Times/$(SERVICE_SYSTEM_CODE)
+
+local-build:
+	obt build --build-folder="./public/" --sass="./src/main.scss" --js="./src/main.js"
+
+production-build:
+	bower install
```

Comment:
We _shouldn't_ need this 🤔

Reply:
`obt install` is an option but would run `npm install` a second time.

Reply:
When these move into the shared Makefile, would it make sense to update the shared install command to use `obt install`?
https://github.com/Financial-Times/origami-service-makefile/blob/master/index.mk#L45

Reply:
Realistically the install happens before we build anyway, doesn't it? In production at least. And it will have to be manual. And I was going to add an error in case it hadn't been built so I'll actually go do that now. 🙈